### PR TITLE
Added docs for using metrics

### PR DESCRIPTION
**Pull request by ziccardi**
## Motivation

JIRA: https://issues.jboss.org/browse/AEROGEAR-2864

## Description

New documentation

## Progress

- [x] Add metrics docs

### CODE REVIEW

File: docs/modules/ROOT/pages/using-metrics.sdk.adoc [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+- *App ID*: The package name of your App
+- *App version*: The version of your App as defined in the Manifest `versionName` tag
+- *SDK version*: The version of the Aerogear SDK that is used
+- *Framework*: Framework used for development (`Xamarin.Android` or `Xamarin.iOS`)
+
+=== Device metrics
+
+- *Platform*: Hardcoded to `android` or `iOS`
+- *Platform Version*: The API level of the device or, on iOS, the  version of the platform (11.4, etc.)
```
Comment (wei-lee) [inline docs/modules/ROOT/pages/using-metrics.sdk.adoc]: Might be better to call out explicitly that the API level is for Android.

**Comment by wei-lee** [comment]
@ziccardi Was trying to contact you this morning about this PR. It looks like all the docs are now moved to  https://github.com/aerogear/mobile-docs/tree/master/modules/ROOT/pages/xamarin. Can you confirm with @finp and move this doc to the right place?  Sorry about the confusion but I only knew about it yesterday.

Thanks.

**Comment by finp** [comment]
Can I ask @ziccardi  @jstaffor  to resolve this. A brief glance at it suggests to me that this provides no info for how a developer would use the metrics service in an app, but does provide useful reference material about how, once metrics is set up in the app (already documented), the app automatically sends data, which can later be viewed in Grafana

**Comment by jstaffor** [comment]
The heading in the file states "This document explains how to use the AeroGear Metrics SDK on the Android platform and integrate it with an App.". The content within does not explain how to use the AeroGear Metrics SDK on the Android platform and as such, my initial thought is that this content is not of much use to docs - sorry for the harsh statement.

@ziccardi - my suggestion is to rewrite this and ping me to review. For inspiration on what this content should look like, take a look at these:

 - https://github.com/aerogear/mobile-docs/blob/master/modules/ROOT/pages/push/proc_sending-a-push-notification.adoc
 - https://github.com/aerogear/mobile-docs/blob/master/modules/ROOT/pages/push/proc_preparing-ups-for-use-with-ios.adoc

**Comment by wei-lee** [comment]
@jstaffor I believe the idea of this doc is to reuse the content we have for other platforms, and modify it for xamarin.

We already have a similar doc for android here: https://github.com/aerogear/mobile-docs/blob/master/modules/ROOT/pages/android/using-metrics-sdk.adoc. 

Would it be ok if we just use the same content and modify it for xamarin? There isn't too much difference for developers, as the metrics data are sent automatically. The doc is really to explain what data is being sent for metrics.

**Comment by ziccardi** [comment]
@jstaffor @finp 

@wei-lee : exactly, I didn't want to do anything new/different for xamarin than what we already did for all the other platforms:

- https://github.com/aerogear/mobile-docs/blob/master/modules/ROOT/pages/ios/using-metrics-sdk.adoc
- https://github.com/aerogear/mobile-docs/blob/master/modules/ROOT/pages/android/using-metrics-sdk.adoc

**Comment by jstaffor** [comment]
@wei-lee , @ziccardi - I (kinda) understand - I'm not fully clued in on what is happening here but it does make sense to use "using-metrics-sdk.adoc." as a starting basis for xamarin particularly if the purpose of the doc is to explain what data is being sent for metrics. 

This location seems correct:
 - https://github.com/aerogear/mobile-docs/tree/master/modules/ROOT/pages/xamarin
 given that the location of "using-metrics-sdk.adoc." is:
 - https://github.com/aerogear/mobile-docs/blob/master/modules/ROOT/pages/android

**Pull request merged**